py/torch_tensorrt/dynamo/partitioning/_global_partitioner.py

```diff
@@ -228,8 +228,22 @@ def partition(
     # Determine partitions based on user specifications and operator support
     # Then, fuse partitions and display overview of supported/unsupported operators
     partitions = partitioner.propose_partitions()
```

Comment:
@narendasan @dheerajperi 
There is an upstream change on CapabilityBasedPartitioner from pytorch:
https://github.com/pytorch/pytorch/commit/62403b57b9810cf47955a101a1d642e244722358
which added the prefix option, I am not sure whether we need to add it.
current: (if prefix is not specified, it uses 'fused_' by default)
`fused_graph = partitioner.fuse_partitions(partitions)`
should I to change to:
`fused_graph = partitioner.fuse_partitions(partitions, prefix="_run_on_acc_")`
I have tested both with change or without change, it both works, the only difference is the graph node name:
 ```
 graph():
   %x : [num_users=1] = placeholder[target=x]
   %_run_on_acc_0 : [num_users=1] = call_module[target=_run_on_acc_0](args = (%x,), kwargs = {})
   return (_run_on_acc_0,)
```

    or

   ```
graph():
   %x : [num_users=1] = placeholder[target=x]
   %fused_0 : [num_users=1] = call_module[target=fused_0](args = (%x,), kwargs = {})
   return (fused_0,)
```